rest/admin-ui-ext/src/main/java/org/keycloak/admin/ui/rest/EffectiveRoleMappingResource.java

```diff
@@ -170,7 +192,11 @@ public final List<ClientRole> listCompositeUsersRoleMappings(@PathParam("id") St
     public final List<ClientRole> listCompositeRealmRoleMappings() {
         auth.roles().requireList(realm);
         final RoleModel defaultRole = this.realm.getDefaultRole();
-        return mapping(o -> o.hasRole(defaultRole)).collect(Collectors.toList());
+        //this definitely does not return what the descriptions says
```

Comment:
@vramik What do you think about this endpoint here? It definitely does not do what the description says (also before my change).

Reply:
I agree the description looks a bit suspicious. Thanks for spotting this. I was trying to find out whether the description or the implementation is off, but I wasn't able to locate correct place from where the endpoint is called from UI.

@edewit could you please comment this one?

Reply:
Sorry, for missing this ping. Yes it is used in the UI

Reply:
@edewit But what is it used for? The descriptions says, it returns all client subroles for a realm role with a given id, the implementation actually returns all client subroles of the default realm role and the given id is not used at all.

Reply:
Right the unused ID is because that way we can make the UI always send one also when no ID is needed it's just to make this a more general implementation.

Reply:
Alright, thanks!